catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlQueryResponse.java

```diff
@@ -114,10 +115,9 @@ public CqlQueryResponse(
                                       logOnce.accept(
                                           "Removed duplicate attribute descriptor(s). For more information:\n"
                                               + "(log:set trace org.codice.ddf.catalog.ui.query.cql)");
-                                      if (LOGGER.isTraceEnabled()) {
-                                        LOGGER.trace(
-                                            "Removed duplicate attribute descriptor.({})", ad1);
-                                      }
+                                      LOGGER.trace(
+                                          "Removed duplicate attribute descriptor.({})",
+                                          LogSanitizer.cleanAndEncode(ad1.toString()));
```

Comment:
❓ Now that we're invoking `toString()` manually on `ad1` should we wrap this in a `if` statement for `isTraceEnabled()` since prior the logging framework handled that for us?